ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3436,6 +3436,56 @@ def test_torch_instance_argmin(
     )
 
 
+# argsort
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="argsort",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        max_num_dims=3,
+        min_dim_size=1,
+        max_dim_size=3,
+        min_value=1,
```

Comment:
There is no need to specify the values range.
Also, you can increase the number of `max_num_dims` to a large value assuming `5` as an example. Also, the axis value should be between `-1` and `0` which means you can specify `min_axis` and `max_axis`

Reply:
still failing a test case, I'll try to resolve it locally and update you.